quickstarts/audit/audit-events-analysis/config.yaml

```diff
@@ -0,0 +1,43 @@
+id: 91b45072-df61-4854-8d3f-76766e75d2a4
```

Comment:
curious why there is an ID here... did this get added by copy paste?

Reply:
I added those, does this cause problems?

Reply:
let me check with @aswanson-nr, I'm not 100% sure how we handle the `IDs` for these config files, maybe they are manually added? 🤔

Reply:
Yes @kidk they are auto-generated as part of the Release workflow [here](https://github.com/newrelic/newrelic-quickstarts/blob/e73caa3ca98272eb27a9b25cf7d7b13964120b37/.github/workflows/release.yml#L32) so if you could remove these that would be great

Reply:
@kidk i went ahead and did this for you 🤠